quipucords/scanner/network/runner/inspect.yml

```diff
@@ -28,4 +28,5 @@
     - insights
     - system_purpose
     - redhat_release
+    - memory
```

Comment:
Just curiosity, any specific reason why you added memory in this exact place?

Reply:
I'd add as the last, but the event processor we attach to ansible runner expects `host_done` to be the last one.